<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadMap</class>
 <widget class="QDialog" name="LoadMap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load map</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons/resourcen.qrc">
    <normaloff>:/icons/fileopen.png</normaloff>:/icons/fileopen.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0" colspan="4">
    <widget class="QCheckBox" name="load_walk_map">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Load walk map</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QCheckBox" name="load_materials">
     <property name="text">
      <string>Load materials</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QCheckBox" name="load_3d_objects">
     <property name="text">
      <string>Load 3d objects</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QCheckBox" name="load_2d_objects">
     <property name="text">
      <string>Load 2d objects</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QCheckBox" name="load_height_map">
     <property name="text">
      <string>Load height map</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="QCheckBox" name="load_tile_map">
     <property name="text">
      <string>Load tile map</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QCheckBox" name="load_lights">
     <property name="text">
      <string>Load lights</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="icon">
      <iconset resource="icons/resourcen.qrc">
       <normaloff>:/icons/gtk-cancel.png</normaloff>:/icons/gtk-cancel.png</iconset>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>62</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0" colspan="4">
    <widget class="QCheckBox" name="load_terrain">
     <property name="text">
      <string>Load terrain</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Ok</string>
     </property>
     <property name="icon">
      <iconset resource="icons/resourcen.qrc">
       <normaloff>:/icons/gtk-apply.png</normaloff>:/icons/gtk-apply.png</iconset>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>50</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QCheckBox" name="load_particles">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Load particles</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons/resourcen.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>LoadMap</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>LoadMap</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
